README.md

```diff
@@ -11,3 +11,7 @@ A repository of deep learning examples.
 ## Examples
 
 - [YoloV5](yolov5/)
+        - `cd yolov5`
+        - `git clone git@github.com:ultralytics/yolov5.git`
+        - Install YoloV5 requirements.txt
+        - Run `python main.py` to download WiderFace data from HuggingFace. You can then use YoloV5 training/inference CLI as expected.
```

Comment:
add `python -m torch.distributed.launch --nproc_per_node 2 train.py --data data/wider_face.yaml --batch-size 32 --epochs 10 --img-size 768 --project runs/train --name wider_face --weights weights/yolov5s.pt --device 0` 

Also reference notebook (and when published the blog post too)

Reply:
I'm not sure I want the README.md to be this detailed. This is kind of why I don't like README's - they're immediately out-of-date as soon as anyone changes anything. The code should just be more self-explanatory.